pkg/client/redhatsso/client.go

```diff
@@ -102,8 +103,16 @@ func (c *rhSSOClient) getCachedToken(tokenKey string) (string, error) {
 	return "", errors.Errorf("failed to retrieve cached token")
 }
 
+func (c *rhSSOClient) GetAcsAPIToken() (string, error) {
+	return c.getToken([]string{"api.iam.acs"})
+}
+
 // GetToken ...
 func (c *rhSSOClient) GetToken() (string, error) {
+	return c.getToken([]string{})
+}
+
+func (c *rhSSOClient) getToken(additionalScopes []string) (string, error) {
```

Comment:
Use ellipsis so we don't have to pass an empty array to the function in `GetToken()`:
```suggestion
func (c *rhSSOClient) getToken(additionalScopes ...string) (string, error) {
```